src/neuroglancer/util/mouse_drag.ts

```diff
@@ -15,21 +15,25 @@
  */
 
 export type RelativeDragHandler = (event: MouseEvent, deltaX: number, deltaY: number) => void;
-export function startRelativeMouseDrag(initialEvent: MouseEvent, handler: RelativeDragHandler) {
+export function startRelativeMouseDrag(initialEvent: MouseEvent,
+                                       handler: RelativeDragHandler,
+                                       finishDragHandler?: RelativeDragHandler ) {
   let {document} = initialEvent.view;
-  let prevScreenX = initialEvent.screenX, prevScreenY = initialEvent.screenY;
+  let prevScreenX = initialEvent.clientX, prevScreenY = initialEvent.clientY;
   let mouseMoveHandler = (e: MouseEvent) => {
-    let deltaX = prevScreenX - e.screenX;
-    let deltaY = prevScreenY - e.screenY;
-    prevScreenX = e.screenX;
-    prevScreenY = e.screenY;
+    let deltaX = prevScreenX - e.clientX;
+    let deltaY = prevScreenY - e.clientY;
+    prevScreenX = e.clientX;
+    prevScreenY = e.clientY;
     handler(e, deltaX, deltaY);
   };
   let button = initialEvent.button;
   let mouseUpHandler = (e: MouseEvent) => {
     if (e.button === button) {
       document.removeEventListener('mousemove', mouseMoveHandler, true);
       document.removeEventListener('mouseup', mouseUpHandler, false);
+
+      if (typeof(finishDragHandler) != 'undefined') finishDragHandler(e, prevScreenX - e.clientX, prevScreenY - e.clientY);
```

Comment:
You can check for undefined here and elsewhere with the shorter syntax finishDragHandler !== undefined.  Note the triple equals.